python/lsst/afw/geom/box.cc

```diff
@@ -31,19 +31,28 @@ using namespace py::literals;
 namespace lsst {
 namespace afw {
 namespace geom {
+namespace {
+
+using PyBox2I = py::class_<Box2I, std::shared_ptr<Box2I>>;
+using PyBox2D = py::class_<Box2D, std::shared_ptr<Box2D>>;
+
+PYBIND11_PLUGIN(box) {
+    py::module mod("box");
+
+    py::object modCoordinates = py::module::import("lsst.afw.geom.coordinates");
```

Comment:
It could be.  But I'm not using any methods besides the ones on `object`, so I figured this was marginally better in the sense that it'd be harder to accidentally add new content to the `coordinates` module this way.